types/logs.go

```diff
@@ -31,8 +33,9 @@ type AccessLog struct {
 
 // AccessLogFilters holds filters configuration
 type AccessLogFilters struct {
-	StatusCodes   StatusCodes `json:"statusCodes,omitempty" description:"Keep access logs with status codes in the specified range" export:"true"`
-	RetryAttempts bool        `json:"retryAttempts,omitempty" description:"Keep access logs when at least one retry happened" export:"true"`
+	StatusCodes   StatusCodes    `json:"statusCodes,omitempty" description:"Keep access logs with status codes in the specified range" export:"true"`
+	RetryAttempts bool           `json:"retryAttempts,omitempty" description:"Keep access logs when at least one retry happened" export:"true"`
+	Duration      parse.Duration `json:"duration,omitempty" description:"Keep access logs with response times above the specific duration" export:"true"`
```

Comment:
Is it possible to use `time.Duration` as type as well? I'm not so much into the `flaeg` stuff. Maybe @ldez knows which type is best to use here. I saw `time.Duration` as example in the flaeg README, but on other places we use `flaeg.Duration` mostly. I'm not sure which one we should use.

Reply:
`flaeg.Duration` is replaced by `parse.Duration`.

`parse.Duration` is the good type for the configuration.